bower.json

```diff
@@ -8,9 +8,7 @@
   "main": "support/entry.client.js",
   "license": "BSD3",
   "dependencies": {
-    "purescript-pux": "^11.0.0",
-    "purescript-css": "^3.0.0",
-    "purescript-aff": "^3.0.0",
-    "purescript-affjax": "^4.0.0"
+    "purescript-pux": "^13.0.0",
+    "purescript-foreign-object": "^1.0.0"
```

Comment:
There seems to be a missing dependency in one of the bower components so including this until the missing dependency is resolved.